### DISCUSSION TITLE
Lobe AI Chat

### DISCUSSION THREAD

**Discussion by x0rb** [coollabsio/coolify#4663] [Service Template Requests]
The most popular chat wrapper https://github.com/lobehub/lobe-chat

**Comment by caglar1** [comment]
we need this app, please integrate it
Thanks

**Reply by goodniceweb** [reply]
Indeed it would be great to have it, especially considering lobe-chat has OAuth, unlike OpenWebUI with only LDAP

**Comment by SeriousM** [comment]
I tried to setup lobechat-db but for me it was quite hard to get all the components configured.

Based on the setup script from lobechat selfhosting advanced deployment - docker compose:
https://lobehub.com/docs/self-hosting/server-database/docker-compose
https://raw.githubusercontent.com/lobehub/lobe-chat/HEAD/docker-compose/local/docker-compose.yml
https://raw.githubusercontent.com/lobehub/lobe-chat/refs/heads/main/docker-compose/local/.env.example

The good part:
- Postgres Vector v17 works very well and can be setup with coolify
- minIO works well and can be setup with coolify
- casdoor is not included with coolify but "authentik" is, which is included. lobechat does support authentik via next-auth (https://lobehub.com/docs/self-hosting/advanced/auth/next-auth/authentik)
- searchnx works well and can be setup with coolify
- lobehub/lobe-chat-database, the main part, is obviously not included with coolify and I don't get it configured well to work with all the required parts.

Maybe someone could help build on top of my findings?

**Comment by jacow** [comment]
Couldn't agree more, Lobe Chat is super cool.

**Comment by volfadar** [comment]
up, almost a year i wanted coolify have lobechat templates, and no one able to do it, like dokploy can have lobe chat, why coolify can't ;))

**Comment by yipfram** [comment]
Hi, I added a PR for the smaller version of Lobechat: https://github.com/coollabsio/coolify/pull/6655
